src/content/developers/tutorials/monitoring-geth-with-influxdb-and-grafana/index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Monitoring Geth with InfluxDB and Grafana
+description: 
+author: "Mario Havel"
+tags: ["clients", "geth", "nodes"]
+skill: intermediate
+lang: en
+sidebar: true
+---
+
+Monitoring your node is a best practice which can help you to be informed about its performance, status, identify potential issues and even alert you if some problem occurs. Client implementations including [Geth](https://geth.ethereum.org/) offer thorough metrics collection and reporting. 
+
+## Prerequisites {#prerequisites}
+
+* Tutorial assumes that you are already running an instance of Geth client. 
+* Most of the steps and examples are for linux environment, basic terminal knowledge will be helpful. 
+* To get familiar with Geth's suite of metrics, its use for debugging and why it might be important to you, check talk [Monitoring an Ethereum infrastructure by Péter Szilágyi](https://www.youtube.com/watch?v=cOBab8IJMYI).
```

Comment:
- Check out this video overview of Geth's suite of metrics: [Monitoring Ethereum infrastructure by Péter Szilágyi](https://www.youtube.com/watch?v=cOBab8IJMYI)